runner/src/simulate.rs

```diff
@@ -695,6 +697,39 @@ impl ScenarioState {
         Ok(metrics.state.anchor_status)
     }
 
+    pub async fn model_sync_status(
+        &self,
+        peer: &Peer,
+        stream_id: String,
+    ) -> Result<bool, anyhow::Error> {
+        let client = reqwest::Client::new();
```

Comment:
creating a new client is heavy (it has an internal threadpool). we should get one of out of the user_data object/a local static/or pass it in as a parameter.

See https://docs.rs/reqwest/latest/reqwest/struct.Client.html specifically `The Client holds a connection pool internally, so it is advised that you create one and reuse it.`

Reply:
I do not see how we can use the GooseUser outside a GooseTransaction. We can create a static client and use that. For now I can create it in simulate.rs